routers/api/v1/api.go

```diff
@@ -1279,7 +1279,10 @@ func Routes() *web.Router {
 				}, reqToken(), reqAdmin())
 				m.Group("/actions", func() {
 					m.Get("/tasks", repo.ListActionTasks)
-					m.Get("/runs/{run}/artifacts", repo.GetArtifactsOfRun)
+					m.Group("/runs/{run}", func() {
+						m.Get("/artifacts", repo.GetArtifactsOfRun)
+						m.Delete("", reqRepoWriter(unit.TypeActions), repo.DeleteActionRun)
```

Comment:
Does it need `reqToken()`?

It's unclear to me why sometimes `reqToken()` is used (above), sometimes it isn't (below).

Reply:
* Some inconsistencies: they were done intentionally, there were some reasons to do so.
* Some other inconsistencies: they were abuses and need to be fixed.

"Others wrote wrong code" does mean that we should follow the wrong code.

----

So, back to this PR: should `reqToken` be used? Or `reqToken` should never be used intentionally?